app/shared/api-services/special-coverage/special-coverage-factory.js

```diff
@@ -101,6 +102,28 @@ angular.module('apiServices.specialCoverage.factory', [
             });
           },
           /**
+           * Load campaign data from Tunic endpoint
+           */
+          $loadTunicCampaign: function () {
+            if (_.isNumber(this.tunicCampaignId)) {
+              return $http.get(TunicConfig.buildBackendApiUrl('campaign/' + this.tunicCampaignId + '/')).then(function (result) {
+                return result.data;
```

Comment:
[These interceptors](https://github.com/theonion/bulbs-cms/search?utf8=%E2%9C%93&q=interceptor&type=Code) currently show that popup in the lower right when a bad response comes back.

I think for now, because this returns a promise interface, it will make it easier to show a meaningful error message in the future.

If we want to do error messages differently, we should really handle that as a separate refactor instead of making another path of code that we have to remove/refactor anyways.